README.md

```diff
@@ -97,6 +97,14 @@ You can pass the following options to `cmake ..`.
 * `-DUNDEFINED_BEHAVIOR_SANITIZER` -- Build with UndefinedBehaviorSanitizer
 * `-DLINK_PROFILER` -- Link with gperftools (available profiler options are listed [here](https://github.com/gperftools/gperftools))
 
+For windows builds, you will have to include additional flags for libwebsockets CMake. Add the following flags to your cmake command, or edit the CMake file in ./CMake/Dependencies/libwebsockets-CMakeLists.txt with the followig:
```

Comment:
Nit: Incorrect spelling of following.

Reply:
correct in one location and my finger skipped an 'n' in another. Fixing.

Reply:
Nit: `w` in windows should be capital.... not blocking